python/lsst/ts/phosim/CloseLoopTask.py

```diff
@@ -753,14 +835,98 @@ def _calcWfErrFromImg(
         # Ingest images into butler gen3
         self.ingestData(butlerRootPath=butlerRootPath, instName=instName)
 
-        listOfWfErr = self.runWep(
+        listOfWfErr = self.runPistonWep(
             extraObsId, intraObsId, butlerRootPath, instName, numPro=numPro
         )
 
         return listOfWfErr
 
-    def runWep(self, extraObsId, intraObsId, butlerRootPath, instName, numPro=1):
-        """Run wavefront estimation pipeline task.
+    def runWfsWep(self, obsId, butlerRootPath, instName, numPro=1):
+        """Run wavefront estimation pipeline task for wavefront sensors.
+
+        Parameters
+        ----------
+        obsId : `int`
+            Observation id.
+        butlerRootPath : `str`
+            Path to the butler gen3 repos.
+        instName : `str`
+            Instrument name.
+        numPro : int, optional
+            Number of processor to run DM pipeline. (the default is 1.)
+
+        Returns
+        -------
+        listOfWfErr : `list` of `SensorWavefrontError`
```

Comment:
Please be consistent that you want to add `` to data type or not (see `numPro`).